### ISSUE TITLE
Vibrate not working in slot A

### ISSUE THREAD

**Issue by nexus382** [flathub/org.flycast.Flycast#202]
Ever since I started playing with fly cast a few months ago I have tried to get the vibrate feature working on both my computer and my retroid pocket 4 with no success. When I have the rumble pack in slot a of device a and my vmu and slot b vibrate does not become available in any game. The other night I was just messing around and for some reason swapped the vmu and the rumble so now the rumble is in slot b and the vmu is in slot a, and I just so happened to be in the options menu of the game, and I noticed the vibrate setting was not grayed out anymore.... It allowed me to flip it on immediately and it worked in game... I tried it in more than one game and this was the only way I could get vibrate to work, the game I last did specifically was four wheel thunder. I did test it on the computer as well and it does seem to work there too now as long as I use The rumble in slot 2 , not 1.

**Comment by flyinghead** [comment]
This isn't the right github repository to report Flycast issues.

However this isn't a Flycast issue: it is physically impossible to connect the original Sega vibration pack in the slot A of a controller. So I suspect many games are expecting it to be in slot B only.

However vibration pack in slot A does work for some games: try Sonic Adventure for example.

**Comment by nexus382** [comment]
For anyone else with this same issue that finds this. DO NOT USE VIBRATE PACK IN SLOT 1,  THE DREAMCAST DID NOT ALLOW THIS, SO MANY GAMES WILL NOT SEE IT SO VIBRATION WONT WORK.

TLDR: VIBRATE PACK ONLY WORK CORRECT IN SLOT 2
